example/mysql/views/layouts/main.tt

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Management of company users</title>
```

Comment:
Can we parameterize this so it shows the correct table/record names if used with a different simple_crud() call?